src/atoms/ErrorMessage/Readme.md

```diff
@@ -0,0 +1,40 @@
+ErrorMessage example:
+
+Simply wrap [`Transition`](#transition) around the `ErrorMessage` and all styling/animation will be handled.
+
+```example
+  class ErrorMessageExample extends React.Component {
+    constructor() {
+      super()
+
+      this.state = { visible: false }
+
+      this.onClick = this.onClick.bind(this);
+    }
+
+    onClick() {
+      this.setState({ visible: !this.state.visible });
+    }
+
+    render() {
+      return (
+        <div>
+          <Button
+            onClick={this.onClick}
+          >
+            Click Me to reveal error message
+          </Button>
+
+          <Transition
+            condition={this.state.visible}
+            component={ErrorMessage}
+          >
+            'Error'
```

Comment:
Why do you need to pass in the children here? I looked at the component and I don't really get it. Are you passing in a component that then wraps itself around the children in the `Transition` component?